src/Frontend/Page/Admin.hs

```diff
@@ -292,8 +292,19 @@ instance ToHtml PageAdminSettingsGaPUsersView where
                             input_ [type_ "text", class_ "inline-search-input", value_ "Nutzersuche"] -- FIXME Placeholder not value
                             a_ [href_ U.Broken, class_ "inline-search-button"] $ i_ [class_ "icon-search"] nil -- FIXME dummy
 
-                let renderUserRow :: forall m. (Monad m) => User -> HtmlT m ()
-                    renderUserRow user = tr_ $ do
+                let renderUserRow :: forall m. (Monad m) => UserView -> HtmlT m ()
+                    renderUserRow (DeletedUser user) = tr_ $ do
+                        td_ $ nil
+                        td_ $ user ^. userLogin . unUserLogin . html
+                        td_ (case user ^. userRole of
+                                Student cl    -> toHtml $ showSchoolClass cl
+                                ClassGuest cl -> toHtml $ showSchoolClass cl
+                                _             -> nil)
```

Comment:
We have `ideaSpaceToSchoolClass` in `Types`.  This here could be a similar helper.  I thought we already had one, but I can't find it any more.